### ISSUE TITLE
Image captcha ResourceWarning: unclosed file

### ISSUE THREAD

**Issue by lukemaxwell** [AdminAnticaptcha/anticaptcha-python#13]
The image captcha `solve_and_return_solution` method opens the image file on line 7:
https://github.com/AdminAnticaptcha/anticaptcha-python/blob/2fe991e0b7322c95d0a3354fab3ad46ede8e884d/anticaptchaofficial/imagecaptcha.py#L7

However the file is never closed.

**Comment by caioluciofc** [comment]
Try using:

`with open(file_path, 'rb') as img:
`

**Comment by lukemaxwell** [comment]
This issue is not about how a user might close a file in python, the issue is that `imagecaptcha.solve_and_return_solution` does not do so. 

As you can see, the method signature defines a `file_path` argument rather than an image object so it is not up to the user to open and close the file. The problem lies with the `solve_and_return_solution` method:

```python
def solve_and_return_solution(self, file_path, **kwargs):
    img = open(file_path, 'rb')
```
The method should be modified to use the with context manager or close the file explicitly.

**Comment by AdminAnticaptcha** [comment]
fixed in version 1.0.31
